stages/cloud-provider/aws/lead/variables.tf

```diff
@@ -30,7 +30,7 @@ variable "asg_max_size" {
 
 variable "instance_types" {
   type    = list(string)
-  default = ["m5.xlarge", "c5.xlarge", "m4.xlarge", "c4.xlarge", "t3.xlarge", "r5.xlarge"]
+  default = ["m5.xlarge", "m4.xlarge", "t3.xlarge", "r5.xlarge"]
```

Comment:
for some reason, the new ASGs only consisted of `c5.xlarge` images.  AWS MNG docs recommend that each node group uses the same instance type (meaning a similar number of vCPUs and memory), so mixing these doesn't work very well.

we can re-add these later if we want different preemptible node groups to use different image types, but the `m` images have a good combination of CPU and memory so I just kept those in for now.